USFMToolsSharp/Models/Markers/MSMarker.cs

```diff
@@ -9,12 +9,16 @@ namespace USFMToolsSharp.Models.Markers
     /// </summary>
     public class MSMarker : Marker
     {
+        public int Weight = 1; 
         public string Heading;
         public override string Identifier => "ms";
         public override string PreProcess(string input)
         {
             Heading = input;
             return string.Empty;
         }

```

Comment:
Also I don't think this is a parent of \mr so it shouldn't be set this way. Because I think we can have \mr without \ms

Reply:
The specs state "The text reference range listed under a major section heading."
I noticed that the examples and id-ayt text always had them together.

I can see a ms without mr, but I can't wrap my head around a mr without ms, because there wouldn't be any idication of a major section.

Reply:
Ok yeah that is right.